cmd/query/main.go

```diff
@@ -122,58 +117,17 @@ func main() {
 			compressHandler := handlers.CompressHandler(r)
 			recoveryHandler := recoveryhandler.NewRecoveryHandler(logger, true)
```

Comment:
looks like L102-118 could also be refactored into `httpserver` pkg, and reused in all-in-one